### ISSUE TITLE
Área do parlamentar: tema ou plugin?

### ISSUE THREAD

**Issue by eduardozulian** [campanhacompleta/prj#7]
Fiquei pensando aqui na ideia do cadastro de parlamentares e eis a minha dúvida: seria interessante criarmos um plugin para cadastro de parlamentares de uma forma genérica a ponto de atender a uma demanda maior?

**Comment by jpmehl** [comment]
Isto pode ser bem legal, termos o "Plugin ParlAberto" dos parlamentares abertos e transparentes, que possam ser utilizados em vários temas....

**Comment by eduardozulian** [comment]
Exato! Poderíamos, inclusive, começar com os campos simples mesmo:

**Dados gerais**
- Nome
- Nome completo do parlamentar
- Data de nascimento
- Naturalidade
- Estado civil
- Ocupação
- Escolaridade
- Biografia
- Links externos (Wikipedia, Facebook, Twitter etc)

**Dados para contato**
- Email
- Endereço
- Telefone

**Dados de transparência**
- Link para prestação de contas ao TRE
- Link para prestação de contas do mandato
- Equipe do mandato

**Comment by eduardozulian** [comment]
Aliás, chamando @jacsonp aqui pra pegar umas ideias.

**Comment by jacsonp** [comment]
tenho aquele esqueleto de post type que podemos usar, enviei sem querer, pq esse esqueleto, nele é fácil criar os campos e já vem com a opção de cadastro no frot

**Comment by jacsonp** [comment]
Posso criar o esqueleto para o plugin baseado no emrede do fluxo, e nele colocamos um template padrão e deixamos a opção para que o tema redesenhe como no delibera
https://github.com/redelivre/fluxo/tree/master/inc/emrede

**Comment by eduardozulian** [comment]
Acho aquela estutura muito gigantesca para um post type, @jacsonp. Eu queria começar algo simples (pra ser disponibilizado até mesmo no repositório oficial do WP) e só depois ir atualizando. Tem umas paradas que seriam legais de ter como o cadastro fácil de metas, mas no geral num sei se precisamos de tudo isso aí.

**Comment by jacsonp** [comment]
sério? Para mim é simples, o que a deixa grande é o filtro de html, que coloquei para melhorar a segurança no front, mas se desabilitarmos o wp_editor do front, da para tirar

**Comment by eduardozulian** [comment]
Conseguimos criar um repositório na com uma versão simplificada daquele código para usarmos como base? Assim teríamos também um lugar específico para fazer melhorias.

**Comment by jpmehl** [comment]
Eu gosto mais da idéia de plugin mesmo. Acho que o esquema de post type é bom e resolve, mas seria massa podermos disponibilizar esta solução, revolveria o problema de uma galera.

**Comment by eduardozulian** [comment]
Pensando em chamar o plugin simplesmente de **parlamentar**. Que tal? Fechando isso, já crio o repositório.

**Comment by jpmehl** [comment]
Manda ficha @eduardozulian
